public/components/CampaignAnalytics/Analytics/CampaignUniquesChart.js

```diff
@@ -9,16 +9,17 @@ export default class CampaignUniquesChart extends React.Component {
 
   render () {
 
-    console.log(this.props.data);
-
-    let data = this.props.data.map( (item) => {
+    let data = [];
+    if (this.props.data) {
```

Comment:
@LATaylor-guardian Just saw your comment a second too late!  I'll do it in another PR